accepted/2020/windows-specific-apis/windows-specific-apis.md

```diff
@@ -0,0 +1,475 @@
+    - `ReceiveFrom(Byte[], Int32, SocketFlags, EndPoint)`
+    - `ReceiveFrom(Byte[], SocketFlags, EndPoint)`
+    - `ReceiveFromAsync(SocketAsyncEventArgs)`
+    - `ReceiveMessageFrom(Byte[], Int32, Int32, SocketFlags, EndPoint, IPPacketInformation)`
+    - `ReceiveMessageFromAsync(SocketAsyncEventArgs)`
+    - `SetIPProtectionLevel(IPProtectionLevel)`
+* **SocketTaskExtensions**
+    - `AcceptAsync(Socket)`
+    - `AcceptAsync(Socket, Socket)`
+    - `ReceiveFromAsync(Socket, ArraySegment<Byte>, SocketFlags, EndPoint)`
+    - `ReceiveMessageFromAsync(Socket, ArraySegment<Byte>, SocketFlags, EndPoint)`
+* **TcpListener**
+    - `AllowNatTraversal(Boolean)`
+* **TransmitFileOptions**
+    - `Disconnect`
+    - `ReuseSocket`
+    - `UseKernelApc`
+    - `UseSystemThread`
+    - `WriteBehind`
+* **UdpClient**
+    - `AllowNatTraversal(Boolean)`
+
+### System.Runtime.InteropServices
+
+* **DispatchWrapper**
+    - `get_WrappedObject()`
+* **Marshal**
+    - `AddRef(IntPtr)`
+    - `BindToMoniker(String)`
+    - `ChangeWrapperHandleStrength(Object, Boolean)`
+    - `CreateAggregatedObject(IntPtr, Object)`
+    - `CreateAggregatedObject<T>(IntPtr, T)`
+    - `FinalReleaseComObject(Object)`
+    - `GetComInterfaceForObject(Object, Type)`
+    - `GetComInterfaceForObject(Object, Type, CustomQueryInterfaceMode)`
+    - `GetComInterfaceForObject<T, TInterface>(T)`
+    - `GetIUnknownForObject(Object)`
+    - `GetNativeVariantForObject(Object, IntPtr)`
+    - `GetNativeVariantForObject<T>(T, IntPtr)`
+    - `GetObjectForIUnknown(IntPtr)`
+    - `GetObjectForNativeVariant(IntPtr)`
+    - `GetObjectForNativeVariant<T>(IntPtr)`
+    - `GetObjectsForNativeVariants(IntPtr, Int32)`
+    - `GetObjectsForNativeVariants<T>(IntPtr, Int32)`
+    - `GetStartComSlot(Type)`
+    - `GetTypedObjectForIUnknown(IntPtr, Type)`
+    - `GetTypeFromCLSID(Guid)`
+    - `GetTypeInfoName(ITypeInfo)`
+    - `GetUniqueObjectForIUnknown(IntPtr)`
+    - `QueryInterface(IntPtr, Guid, IntPtr)`
+    - `Release(IntPtr)`
+    - `ReleaseComObject(Object)`
+
+### System.Security.Cryptography
+
+* **CspKeyContainerInfo**
+    - `get_Accessible()`
+    - `get_Exportable()`
+    - `get_HardwareDevice()`
+    - `get_KeyContainerName()`
+    - `get_KeyNumber()`
+    - `get_MachineKeyStore()`
+    - `get_Protected()`
+    - `get_ProviderName()`
+    - `get_ProviderType()`
+    - `get_RandomlyGenerated()`
+    - `get_Removable()`
+    - `get_UniqueKeyContainerName()`
+* **DSACryptoServiceProvider**
+    - `get_CspKeyContainerInfo()`
+    - `ImportCspBlob(Byte[])`
+* **PasswordDeriveBytes**
+    - `CryptDeriveKey(String, String, Int32, Byte[])`
+* **RC2CryptoServiceProvider**
+    - `set_UseSalt(Boolean)`
+* **RSACryptoServiceProvider**
+    - `.ctor(CspParameters)`
+    - `.ctor(Int32, CspParameters)`
+    - `get_CspKeyContainerInfo()`
+    - `ImportCspBlob(Byte[])`
+
+### System.Security.Cryptography.X509Certificates
+
+* **X509Chain**
+    - `.ctor(IntPtr)`
+
+### System.ServiceModel
```

Comment:
Someone familiar with WCF will need to comment on this section.

Reply:
We have zero api's which are Windows specific.